pkg/connector/group.go

```diff
@@ -122,10 +123,8 @@ func (o *groupResourceType) Grants(ctx context.Context, resource *v2.Resource, p
 		if err != nil {
 			return nil, "", nil, err
 		}
-		grant := sdkGrant.NewGrant(resource, groupMemberEntitlement, gmID)
-		annos := annotations.Annotations(grant.Annotations)
-		annos.Update(v1Identifier)
-		grant.Annotations = annos
+		grant := sdkGrant.NewGrant(resource, groupMemberEntitlement, gmID, sdkGrant.WithAnnotation(v1Identifier))
+		grant.Id = member.Id
```

Comment:
Is this going to cause churn, or is this what the grant ID was before?

Reply:
See PR description for more ;)